docs/contribute.html

```diff
@@ -106,8 +106,9 @@ <h2 class="display-4">Legal Terms</h2>
     <!-- Footer -->
     <footer class="py-3 bg-dark">
       <div class="container">
-        <p class="m-0 text-center text-white">Copyright &copy; RaiderPlanner 2017</p>
+        <p class="m-0 text-center text-white">Copyright &copy; <a href="../AUTHORS.txt">RaiderPlanner Contributors</a></p>
```

Comment:
This should still have the year, 2017. Also, note that the file in the repository is simply called AUTHORS, with no extension. Clicking on the link results in the file not being found.